docs/index.asciidoc

```diff
@@ -142,6 +142,8 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-query>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-query_template>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-result_size>> |<<number,number>>|No
+| <<plugins-{type}s-{plugin}-retry_on_failure>> |<<number,number>>|No
+| <<plugins-{type}s-{plugin}-result_on_status_>> |<<number,number list>>|No
```

Comment:
I'm guessing this was supposed to be "**retry**_on_status"?  There's also a random `_` at the end of the entry , which is enough to fail docs-ci.  :-(